docs/src/pages/components/slider/UnstyledSlider.js

```diff
@@ -47,19 +46,18 @@ const StyledSlider = styled(SliderUnstyled)(
     border-radius: 50%;
     outline: 0;
     border: 2px solid currentColor;
-    background-color: ${theme.palette.getContrastText(theme.palette.primary.main)};
+    background-color: white;
 
     :hover,
     &.Mui-focusVisible {
-      box-shadow: 0 0 0 0.25rem ${alpha(theme.palette.primary.main, 0.15)};
+      box-shadow: 0 0 0 0.25rem ${alpha('#1976d2', 0.15)};
```

Comment:
Maybe we need to adapt the color based on the dark vs. light mode?

Reply:
True, let me update